pkgs/development/libraries/lyra/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, installShellFiles, meson, ninja }:
+
+stdenv.mkDerivation rec {
+  pname = "lyra";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "bfgroup";
+    repo = "lyra";
+    rev = version;
+    sha256 = "1wcwsmg41bmjir6pjrjxrwccqj25d9068ifi9m6xz6q3fhaq6s81";
+  };
+
+  nativeBuildInputs = [ meson ninja ];
+
+  enableParallelBuilding = true;
+
+  patchPhase = "sed -i s#/usr#$out#g meson.build";
```

patchPhase does a lot of work, better to use the pre/post hooks
```suggestion
  postPatch = "sed -i s#/usr#$out#g meson.build";
```